src/jit/gentree.cpp

```diff
@@ -13443,3 +13443,100 @@ void ReturnTypeDesc::Initialize(Compiler* comp, CORINFO_CLASS_HANDLE retClsHnd)
 
 #endif // FEATURE_UNIX_AMD64_STRUCT_PASSING
 }
+
+//-------------------------------------------------------------------
+// GetABIReturnReg:  Return ith return register as per target ABI
+//
+// Arguments:
+//     idx   -   Index of the return register
+//
+// Return Value:
+//     Returns ith return register as per target ABI.
+//
+// Notes:
+//     Right now this is implemented only for x64 Unix
+regNumber ReturnTypeDesc::GetABIReturnReg(unsigned idx)
+{
+    assert(idx > 0);
```

Comment:
Why is this 1-based? Most things in the JIT are zero-based, so this seems quite awkward.